proposals/34:upgrade-10/package.json

```diff
@@ -8,6 +8,7 @@
     "type": "module",
     "license": "Apache-2.0",
     "dependencies": {
+        "@agoric/synthetic-chain": "file:./agoric-synthetic-chain-0.0.1-alpha.tgz",
```

Comment:
There's a copy of this tar file in each proposals directory? I guess that makes sense...

Reply:
Once we're publishing to NPM then it can just pull from there. This is a stop gap